src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -12,7 +12,7 @@
         ],
         "responses": {
           "200": {
-            "description": "Success"
+            "description": "OK"
```

Comment:
Swashbuckle uses custom mappings for it's descriptions. We use the `ReasonPhrases` API.